Misc/NEWS.d/next/Library/2017-12-19-09-23-46.bpo-32373.8qAkoW.rst

```diff
@@ -0,0 +1 @@
+Add socket.getblocking() method.
```

Comment:
It seems like you modified settimeout() behaviour. It must be documented here and in settimeout() doc.

Reply:
I'm wrong, settimeout() is (no longer) modified by this PR.

My comment was for a previous version of the PR. The PR evolved too quickly, I was lost, sorry.